tests/gis_tests/rasterapp/models.py

```diff
@@ -1,9 +1,34 @@
-from django.contrib.gis.gdal import HAS_GDAL
 from django.contrib.gis.db import models
+from django.contrib.gis.gdal import HAS_GDAL
+from django.db import connections
+
+
+class SkipChecksIfNotDBFeaturesMixin(object):
```

Comment:
It seems like this could be useful in other tests at some point. Rather than burying it here, it might be useful to put it somewhere more reusable/discoverable (even if it's not public API at this point). Maybe `django.core.checks.tests.py`? Any opinion @charettes?